src/renderer/components/about-app.tsx

```diff
@@ -188,6 +191,9 @@ export default class AboutApp extends React.Component<{}, IState> {
    * Callback to handle event when a component is mounted
    */
   public componentDidMount(): void {
+    setTimeout(() => {
+      this.copyButtonRef.current?.blur();
```

Comment:
Same, this can be merged but Windows is always unfaithful to timing like this? Just a Question @sbenmoussati